### ISSUE TITLE
nodejs4.3 no longer supported - stack fails to set-up

### ISSUE THREAD

**Issue by ashantyk** [AshanFernando/SESEmailForward#3]
while preparing the stack, aws stops the process with an error saying that nodejs4.3 is no longer supported and you should upgrade to 8.1

exact error:

The runtime parameter of nodejs4.3 is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs8.10) while creating or updating functions. (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: 5f164ac7-1e89-11e9-b1bc-e9d7c3aacfd1)
--

**Comment by ScientificProgrammer** [comment]
I had the same issue. I was able to get the CloudFormation stack to build by updating the [forwarder.template](https://github.com/AshanFernando/SESEmailForward/blob/master/forwarder.template) file by changing the node.js specification from nodejs4.3 to nodejs8.10 and uploading the creation template manually using the CloudFormation console. I've also created a pull request here on this repo.
